l1-contracts/src/core/RollupCore.sol

```diff
@@ -118,23 +119,12 @@ contract RollupCore is
   /*                          CHEAT CODES START HERE                            */
   /* -------------------------------------------------------------------------- */
 
-  function setEpochVerifier(address _verifier) external override(ITestRollup) onlyOwner {
-    CheatLib.setEpochVerifier(_verifier);
-  }
-
-  function setVkTreeRoot(bytes32 _vkTreeRoot) external override(ITestRollup) onlyOwner {
-    CheatLib.setVkTreeRoot(_vkTreeRoot);
-  }
-
-  function setProtocolContractTreeRoot(bytes32 _protocolContractTreeRoot)
-    external
-    override(ITestRollup)
-    onlyOwner
-  {
-    CheatLib.setProtocolContractTreeRoot(_protocolContractTreeRoot);
-  }
-
   function updateManaTarget(uint256 _manaTarget) external override(ITestRollup) onlyOwner {
```

Comment:
Getting rid of the `ITestRollup` would be good in the same go. It can go into the core as it manipulates storage 🫡